src/connector/src/sink/starrocks.rs

```diff
@@ -125,8 +127,8 @@ impl StarrocksSink {
         starrocks_columns_desc: HashMap<String, String>,
     ) -> Result<()> {
         let rw_fields_name = self.schema.fields();
-        if rw_fields_name.len().ne(&starrocks_columns_desc.len()) {
-            return Err(SinkError::Starrocks("The length of the RisingWave column must be equal to the length of the starrocks column".to_string()));
+        if rw_fields_name.len().gt(&starrocks_columns_desc.len()) {
```

Comment:
Use `>` instead of gt.

If we do partial update, should we ensure that the starrrocks pk columns are all included in the sink columns?

Reply:
Now it is time to calibrate whether the PK of rw and sr are completely consistent, and there is a tendency not to make any modifications. If it is changed to SR and rw not being completely consistent, it may lead to accuracy issues with the update. Our other sinks also follow a similar logic

Reply:
Do you mean we should consider the case that the downstream pk of SR is changed after the sink is created? I don't think this is what we should consider, because we are not expected to adapt to downstream pk change yet, and users should be required not to change the pk.

> Our other sinks also follow a similar logic

This is not ture. For downstreams that really have pk (e.g. jdbc sink), we do check whether the pks specified in the with options match the downstream pk.

Reply:
I'm just saying that we have code to check PK, but our verification is an exact match, not a superset like other columns（rw.columns.len ＜＝ sr.columns.len）